### PR TITLE
openpyxl: Updates openpyxl.utils.cell stubs

### PR DESCRIPTION
**Pull request by XF-FW**
Updates this stub file, whose types were not filled in.

### CODE REVIEW

File: stubs/openpyxl/openpyxl/utils/cell.pyi [hunk 1]
```diff
@@ -9,17 +9,17 @@ ABSOLUTE_RE: Any
...
-def range_boundaries(range_string): ...
+def get_column_letter(idx: int) -> str: ...
+def column_index_from_string(str_col: str) -> int: ...
+def range_boundaries(range_string: str) -> tuple[int, int, int, int]: ...
 def rows_from_range(range_string) -> Generator[Any, None, None]: ...
 def cols_from_range(range_string) -> Generator[Any, None, None]: ...
-def coordinate_to_tuple(coordinate): ...
-def range_to_tuple(range_string): ...
-def quote_sheetname(sheetname): ...
+def coordinate_to_tuple(coordinate: str) -> tuple[int, str]: ...
```
Comment (AlexWaygood) [inline stubs/openpyxl/openpyxl/utils/cell.pyi:23]: The second element in the returned tuple is a value in the `_COL_STRING_CACHE` dictionary, which has `str`s for keys and `int`s for values.

https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/utils/cell.py#L202

```suggestion
def coordinate_to_tuple(coordinate: str) -> tuple[int, int]: ...
```

**Review by AlexWaygood — COMMENTED** [review]
Thanks! Just one thing I noticed, otherwise this looks good.

File: stubs/openpyxl/openpyxl/utils/cell.pyi (hunk 1 above)
Comment (XF-FW) [reply stubs/openpyxl/openpyxl/utils/cell.pyi]: Ah nice catch. Thank you!

I've applied your suggestion.

**Review by AlexWaygood — APPROVED** [review]
Looks great!

**Pull request merged**